storio-common-annotations-processor/src/test/java/com/pushtorefresh/storio/common/annotations/processor/introspection/AnnotatedFieldValidationTest.java

```diff
@@ -30,15 +30,15 @@ public void failIfEnclosingElementIsNotType() {
         stub.processor.validateAnnotatedFieldOrMethod(stub.field);
     }
 
-    @Test
-    public void failIfThereIsNoAnnotation() {
-        AnnotationProcessorStub stub = AnnotationProcessorStub.newInstance();
-        when(stub.enclosingElement.getAnnotation(AnnotationProcessorStub.TestClassAnnotation.class)).thenReturn(null);
-
-        expectedException.expect(ProcessingException.class);
-        expectedException.expectMessage("Please annotate class TestClass with TestClassAnnotation");
-        stub.processor.validateAnnotatedFieldOrMethod(stub.field);
-    }
+//    @Test
```

Comment:
To make test work we can mock superclass filed in `AnnotationProcessorStub` constructor:
```java
        TypeMirror fieldSuperClass = mock(TypeMirror.class);
        when(enclosingElement.getSuperclass()).thenReturn(fieldSuperClass);

        ProcessingEnvironment processingEnvironment = mock(ProcessingEnvironment.class);
        Types types = mock(Types.class);
        when(types.asElement(fieldSuperClass)).thenReturn(mock(Element.class));
        when(processingEnvironment.getTypeUtils()).thenReturn(types);
        doCallRealMethod().when(processor).init(processingEnvironment);
        processor.init(processingEnvironment);
```
Test with so much mocking are cut off from reality, but better than nothing =(